### ISSUE TITLE
Create comparison chart of jenni and phenny. 

### ISSUE THREAD

**Issue by ghost** [myano/jenni#85]
Someone walking into a store. They know they want a bot, but they are not sure what brand they want. The salesmen are on coffee break for the 10th time today and now the customer has search for the differences between two products. Now the customer is getting mad cause he or she can't find the comparison chart for the two bots. Now the customer just gets mad and leaves the store only to go to another store and gets a Supybot. 

tl;dr .. would be nice to be able to compare features of the two bots.
